src/invoices/InvoiceForm/FieldBatchGroup/FieldBatchGroup.js

```diff
@@ -7,35 +7,44 @@ import {
 } from '@folio/stripes-acq-components';
 
 import { getBatchGroupsOptions } from '../../../common/utils';
+import BatchGroupValue from '../../InvoiceDetails/BatchGroupValue';
 
-const FieldBatchGroup = ({ batchGroups, isEditPostApproval }) => {
+const FieldBatchGroup = ({ batchGroups, isNonIteractive, batchGroupId }) => {
   const batchGroupsOptions = useMemo(() => {
     return getBatchGroupsOptions(batchGroups);
   }, [batchGroups]);
 
   const labelId = 'ui-invoice.invoice.details.information.batchGroup';
-  const isDisabled = isEditPostApproval || batchGroups.length === 1;
 
   return (
-    <FieldSelectionFinal
-      dataOptions={batchGroupsOptions}
-      disabled={isDisabled}
-      id="invoice-batch-groups"
-      labelId={labelId}
-      name="batchGroupId"
-      required
-      validate={validateRequired}
-    />
+    isNonIteractive || batchGroups.length === 1
+      ? (
+        <BatchGroupValue
+          id={batchGroupId}
+          label={labelId}
+        />
+      )
+      : (
+        <FieldSelectionFinal
+          dataOptions={batchGroupsOptions}
```

Comment:
there is a PR for FieldSelectionFinal for nonInteractive mode https://github.com/folio-org/stripes-acq-components/pull/345/files